serve/triton/to_triton_models.py

```diff
@@ -194,6 +194,36 @@ class ModelToTriton(ToTriton):
 
     Convert model to trt, torchscript or onnx.
     '''
+    def __init__(self, op, model_root, model_name, model_optimize):
+        super().__init__(op.model, model_root, model_name)
+        self._model_optimize = model_optimize
+
     def _prepare_model(self):
-        self._backend = 'tensorrt'
+        succ = False
+        for optimize in self._model_optimize:
+            if optimize in self._obj.model.optimizes:
+                if optimize == 'onnx':
+                    succ = self._obj.save_model(optimize, self._triton_files.onnx_model_file)
+                elif optimize == 'tensorrt':
+                    succ = self._obj.save_model(optimize, self._triton_files.trt_model_file)
+                else:
+                    logger.error('Unkown optimize %s' % optimize)
+                    continue
+                self._backend = optimize
```

Comment:
what if there is no any match between self._model_optimize and self._obj.model.optimizes?

Reply:
and the names of 'self._model_optimize' and 'self._obj.model.optimizes' are confusing. suggest to use proper names such as self._model_format_priority, self._obj.model.supported_formats

Reply:
> what if there is no any match between self._model_optimize and self._obj.model.optimizes?

Return errors, and stop building.